packages/fxa-customs-server/lib/email_record.js

```diff
@@ -18,7 +18,9 @@ module.exports = function(limits, now) {
   EmailRecord.parse = function(object) {
     var rec = new EmailRecord();
     object = object || {};
-    rec.bk = object.bk; // timestamp when the account was banned
+    rec.bk = object.bk; // timestamp when the account was blocked
+    rec.su = object.su; // timestamp when the account was suspected
+    rec.di = object.di; // timestamp when the account was disabled
```

Comment:
Storing the `disable` flag in memcached is problematic. As specced, it's meant to be a permanent ban but for this first cut I'm just sticking it in memcached with everything else. The memcached eviction policy means we need to do something more permanent than that.

Ideally that means using a reliable data store, but I'm conscious there's been talk of getting rid of or re-implementing the customs server. So I wasn't sure we wanted to make the effort to add Redis or MySQL now, if we're going to replace the whole thing eventually anyway.

I did consider a workaround of emitting some kind of heartbeat of all the `disable` states to the logs at regular intervals, to allow those messages to be replayed if evictions are detected. But that's complicated by the fact our records are all keyed by email/IP address, meaning we'd have to iterate every single record to build up the heartbeat state. We could fix *that* by maintaining an inverted index just for the disables, but then we're back to the question of where to store the inverted index. If we put it in memcached it can be evicted, and if we add Redis or whatever then we may as well just skip the whole heartbeat idea and store the disables in Redis.

Suggestions welcome.

Reply:
If `su` or `di` are not available, are there any downstream repercussions? I see you check whether `this.su` or `this.di` have values below before comparing against them so I assume all is fine?

Reply:
> I see you check whether `this.su` or `this.di` have values below before comparing against them so I assume all is fine?

Correct.